CRITICAL - 2023-03-05 08:35:09 --> Invalid file: "v_peta.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('v_peta.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('v_peta', [], true)
 3 APPPATH\Controllers\Home.php(9): view('v_peta')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-05 08:43:37 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension()
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(83): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2023-03-05 08:44:39 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension()
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(83): CodeIgniter\CodeIgniter->initialize()
ERROR - 2023-03-05 09:06:27 --> Error connecting to the database: ErrorException: pg_connect(): Unable to connect to PostgreSQL server: FATAL:  password authentication failed for user "postgres" in C:\xampp\htdocs\petalokasiobjek\system\Database\Postgre\Connection.php:75
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_connect(): U...', 'C:\\xampp\\htdocs...', 75)
#1 C:\xampp\htdocs\petalokasiobjek\system\Database\Postgre\Connection.php(75): pg_connect('host=localhost ...')
#2 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(391): CodeIgniter\Database\Postgre\Connection->connect(false)
#3 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SELECT "table_n...')
#5 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\petalokasiobjek\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\petalokasiobjek\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\petalokasiobjek\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\petalokasiobjek\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\petalokasiobjek\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\petalokasiobjek\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}
ERROR - 2023-03-05 09:07:44 --> Error connecting to the database: ErrorException: pg_connect(): Unable to connect to PostgreSQL server: FATAL:  password authentication failed for user "postgres" in C:\xampp\htdocs\petalokasiobjek\system\Database\Postgre\Connection.php:75
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_connect(): U...', 'C:\\xampp\\htdocs...', 75)
#1 C:\xampp\htdocs\petalokasiobjek\system\Database\Postgre\Connection.php(75): pg_connect('host=localhost ...')
#2 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(391): CodeIgniter\Database\Postgre\Connection->connect(false)
#3 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SELECT "table_n...')
#5 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\petalokasiobjek\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\petalokasiobjek\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\petalokasiobjek\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\petalokasiobjek\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\petalokasiobjek\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\petalokasiobjek\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}
ERROR - 2023-03-05 09:13:20 --> Error connecting to the database: ErrorException: pg_connect(): Unable to connect to PostgreSQL server: FATAL:  password authentication failed for user "postgres" in C:\xampp\htdocs\petalokasiobjek\system\Database\Postgre\Connection.php:75
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_connect(): U...', 'C:\\xampp\\htdocs...', 75)
#1 C:\xampp\htdocs\petalokasiobjek\system\Database\Postgre\Connection.php(75): pg_connect('host=localhost ...')
#2 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(391): CodeIgniter\Database\Postgre\Connection->connect(false)
#3 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SELECT "table_n...')
#5 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\petalokasiobjek\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\petalokasiobjek\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\petalokasiobjek\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\petalokasiobjek\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\petalokasiobjek\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\petalokasiobjek\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}
ERROR - 2023-03-05 09:15:41 --> Error connecting to the database: ErrorException: pg_connect(): Unable to connect to PostgreSQL server: FATAL:  password authentication failed for user "postgres" in C:\xampp\htdocs\petalokasiobjek\system\Database\Postgre\Connection.php:75
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_connect(): U...', 'C:\\xampp\\htdocs...', 75)
#1 C:\xampp\htdocs\petalokasiobjek\system\Database\Postgre\Connection.php(75): pg_connect('host=localhost ...')
#2 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(391): CodeIgniter\Database\Postgre\Connection->connect(false)
#3 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SELECT "table_n...')
#5 C:\xampp\htdocs\petalokasiobjek\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\petalokasiobjek\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\petalokasiobjek\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\petalokasiobjek\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\petalokasiobjek\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\petalokasiobjek\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\petalokasiobjek\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}
